src/sections/Why.vue

```diff
@@ -1,16 +1,228 @@
 <template>
   <Section id='why' as='section'>
-    owo
+    <div class="why__main">
+      <Typography class='why__heading' type='heading2' color='light-yellow' transform='uppercase' as='h2'>
+        Why get involved
+        <img class='why__bubble' src="../assets/Why/grow-bubble.svg">
+      </Typography>
+    </div>
+    <div class='why__card-wrapper'>
+      <div class='why__cards'>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Want to land your
+              next internship?
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              Hackathons are an amazing place to meet
+              mentors and industry professionals in the
+              tech community. A pandemic won't stop us
+              from fostering important conversations.
+            </Typography>
+            <img class="why__laptops" src="../assets/Why/internship/laptops.svg">
+        </div>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Looking to learn from
+              experts?
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              We value sharing knowledge and applying the
+              things we learned. We'll host live workshops
+              all weekend to give you the inspiration you
+              need to get your project off the ground.
+            </Typography>
+            <img class="why__gears" src="../assets/Why/learn/gears.svg">
+        </div>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Want to be rewarded
+              for your hard work?
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              With $38,000 worth of prizes, there’s
+              something for everyone.
+            </Typography>
+            <img class="why__prize" src="../assets/Why/reward/prize.svg">
```

Comment:
Super minor (don't bother changing this). But you can do `@/assets/Why/reward/prize.svg` instead, which is better since if we decide to move this component after in the folders - nothing breaks